src/name_bazaar/ui/events/offering_requests_events.cljs

```diff
@@ -10,7 +10,7 @@
     [medley.core :as medley]
     [name-bazaar.shared.utils :refer [parse-offering-request top-level-name?]]
     [name-bazaar.ui.constants :as constants :refer [default-gas-price interceptors]]
-    [name-bazaar.ui.utils :refer [namehash normalize path-for update-search-results-params]]
+    [name-bazaar.ui.utils :as utils :refer [namehash normalize path-for update-search-results-params]]
```

Comment:
Just a small thing. Please don't use `utils` alias, as there's many different utils files across the app. This can create chaos. Just refer function you need to use.